### ISSUE TITLE
Trouble reading a large set of morphologies

### ISSUE THREAD

**Issue by liesbethvanherpe** [BlueBrain/NeuroM#204]
I have trouble reading a large set of morphologies in .h5 format.

```python
from neurom import ezy
nrns = ezy.load_neurons('/folder_with_2000_morphologies_h5/')
```

results in `"IOError: unable to open file (File accessibilty: Unable to open file)"`

I tried reading the files one by one with

```python
for i in range(2000):
    print i
    nrn = ezy.load_neurons('/folder_with_2000_morphologies_h5/morphology' + str(i))
```

and it seems I can read about 1016 files. The problem is not with a specific file (I checked); reading them in a different order stops at the same number of files.

Is it possible that files are opened but not closed?

**Comment by juanchopanza** [comment]
@liesbethvanherpe Can you check that after fix #205 you can read all the files?

**Comment by liesbethvanherpe** [comment]
I can, thanks!

**Comment by juanchopanza** [comment]
Fixed by #205.
